src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -15547,14 +15547,14 @@ void WriteLoadComponentSummaryTables(EnergyPlusData &state)
             for (int iAirLoop = 1; iAirLoop <= NumPrimaryAirSys; ++iAirLoop) {
                 zoneToAirLoopCool = 0;
                 zoneToAirLoopHeat = 0;
-                if (FinalSysSizing(iAirLoop).CoolingPeakLoadType == DataSizing::SensibleCoolingLoad) {
+                if (FinalSysSizing(iAirLoop).LoadSizeType == DataSizing::Sensible) {
```

Comment:
Not sure if I should keep making these comments or just leave these to a future epJSON pass, but this should also be an `enum class`.

Reply:
Agreed. Wanted to document CI diffs before making that change.